docs/quickstart/go.md

```diff
@@ -38,7 +38,7 @@ $ go get google.golang.org/grpc
 Install the protoc compiler that is used to generate gRPC service code. The simplest way to do this is to download pre-compiled binaries for your platform(`protoc-<version>-<platform>.zip`) from here: [https://github.com/google/protobuf/releases](https://github.com/google/protobuf/releases)
```

Comment:
Maybe change this as the user won't be using a pre-compiled binary, they'll be building the C++ one:

First install the standard C++ implementation of protocol buffers from [https://github.com/google/protobuf/releases](https://github.com/google/protobuf/releases).